nav2_mppi_controller/include/nav2_mppi_controller/tools/utils.hpp

```diff
@@ -266,7 +266,7 @@ inline bool withinPositionGoalTolerance(
 template<typename T>
 auto normalize_angles(const T & angles)
 {
-  auto && theta = xt::eval(xt::fmod(angles + M_PI, 2.0 * M_PI));
+  const auto theta = xt::eval(xt::fmod(angles + M_PI, 2.0 * M_PI));
```

Comment:
Idk if this is actually right, but is solved a 15,000 line puking of templates in `xtensor`. I couldn't find root cause on this warning/error.